frontend/packages/knative-plugin/src/components/sink-source/SinkSourceModal.tsx

```diff
@@ -33,16 +33,23 @@ const SinkSourceModal: React.FC<Props> = ({
   const dirty =
     values?.formData?.sink?.name !== initialValues.formData.sink.name ||
     values?.formData?.sink?.uri !== initialValues.formData.sink.uri;
+
+  // For translation assistance we separate this from the direct return to help give contextual understanding
+  const sinkUriResourceGroup = (
+    <FormSection fullWidth>
+      <SinkUriResourcesGroup namespace={namespace} isMoveSink />
+    </FormSection>
+  );
   return (
     <form className="modal-content modal-content--no-inner-scroll" onSubmit={handleSubmit}>
       <ModalTitle>{t('knative-plugin~Move sink')}</ModalTitle>
       <ModalBody>
-        <p>
-          {t('knative-plugin~Connects')} <strong>{resourceName}</strong> {t('knative-plugin~to')}
-        </p>
-        <FormSection fullWidth>
-          <SinkUriResourcesGroup namespace={namespace} isMoveSink />
-        </FormSection>
+        <Trans ns="knative-plugin">
+          <p>
+            Connects <strong>{{ resourceName }}</strong> to
+          </p>
+          {sinkUriResourceGroup}
```

Comment:
should we have `sinkUriResourceGroup` out of `Trans`? pseudo localization shows as below

![image](https://user-images.githubusercontent.com/5129024/126649364-e7528570-40e6-48e3-b040-a1d18d65edf7.png)

Reply:
Hmmm an interesting point. Not sure this is a "real" problem. I think `[ ... ]` is added by the pseudo translation. And thus won't be part of the normal translations. Let me poke around a bit and see if I can fake a translation or something to see if it's a real issue. Good catch though 👍 

We need it IN the Trans wrapper because it helps the translators understand "what the to" goes to lol. The bug here is we lacked context and separated the items. We are trying to undo that.

Reply:
sure, let me know ...